brokers.yml

```diff
@@ -207,9 +207,7 @@ components:
           schema:
             type: object
             required:
-            - referenceId
             - amount
-            - amountToRefinance
```

Comment:
I think right now we should not touch the broker requirements.
We need to know their reference id 100% because is the one that allows the link between our and their application ;)

Also this flow is slightly different from the partners one :wink: 

I suggest to reset the last commit and keep only the first one :wink: